### 1. Installing the environment

To install the environment:
```
conda env create -f environment.yml
```

### 2. Testing in-silico Polymerase Chain Reaction (isPCR)

Run the command below:
```
./q1.py
```

**Expected output**
The expected output from the provided "q1.py" script is something like as follows:
>Pseudomonas_aeruginosa_PAO1_NC_002516.2:635141-635853 Pseudomonas aeruginosa PAO1, complete genome
TCTCGGCGACGGTCAGCTCGACCTCGCTTTCCAGGGCCGCCAGCTTCTGCTGGTTGCGCAGGATGTCGTCGCGCAGGCGCTCGATGGCCTCGGCGTACTTCGGCTTGCTCTTCAGGACGCTGTCGACCCACTTCTCGTCGGTCTCGTGGTTCGGGAACAGGCGCAGGAAGTCGGCACGCGGCATGCGCGCGTCACGCACGCAGAGCTGCATGATGGCGCGTTCCTGGGCGCGCACGCCTTCCAGGGCGGAGCGCACGCGGGCGACCAGGGCGTCGAACTGCTTGGGCACCAGCTTGATCGGCATGAACAGCTCGGCCAGGCCGGTGAGTTCGGCGGTGGCCTGCTTGCTGCCGCGACCGTGCTTCTTCAGGGCCTTCTTGGCCTTGTCGAGCTGCTCGGAGACCGCGGTGAAACGCAGGCGGGCTTCTTCCGGATCCGGACCGCCGTCGCCTTCGTCGTCGCTGTCGCTGCTGTCGTCGCTTTCTTCTTCCTCGTCGTCCTTCTCTTTCGAGTCGGCGGAATCGTCCTTCAGGTTGACCGGCTCCACCTCTTCGGCGGGCAGGCTGCCGTCATCGGGATCGATATAGCCGCTGAGGACGTCGGAGAGGCGACCGCCTTCGGCGACGATGCGATTGTAGTCGGCCAGGATGCTGTCCACCGTGCCCGGGAACTGGGCGATGGCGCTCATCACTTCGCGGATGCCTTCCTCGATG


### 3. Testing the Needleman-Wunsch global sequence alignment algorithm

Run the command below:
```
./q2.py
```

**Expected output**
The expected output from the provided "q2.py" script is something like as follows:

>CTTCTCGT-CGGTCTCGTGGTTCGGGAAC <br>
>CTT-TCATCCACT-TCGTTGCCCGGGAAC <br>
>True <br>
>11

### 4. Running the amplicon aligner

Run the command below:
```
./amplicon_align.py -h
```
**Usage:**
>amplicon_align.py [-h] -1 ASSEMBLY1 -2 ASSEMBLY2 -p PRIMERS -m MAX_AMPLICON_SIZE --match MATCH --mismatch MISMATCH --gap GAP
